docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -691,28 +761,133 @@ workflows:
       - semgrep-scan
 ```
 
+</TabItem>
+<TabItem value='circleci-standalone'>
+
+```yaml
+version: 2.1
+jobs:
+  semgrep-scan:
+    parameters:
+      default_branch:
+        type: string
+        default: main
+    environment:
+      SEMGREP_RULES: p/default
+
+      # Uncomment the following line to scan changed 
+      # files in PRs or MRs (diff-aware scanning): 
+      # - export SEMGREP_BASELINE_REF = "origin/main"
+      # - git fetch origin "+refs/heads/*:refs/remotes/origin/*"
+      # SEMGREP_BASELINE_REF: << parameters.default_branch >>
+
+    docker:
+      - image: returntocorp/semgrep
+    steps:
+      - checkout
+      - run:
+          name: "Semgrep scan"
+          command: semgrep ci
+workflows:
+  main:
+    jobs:
+      - semgrep-scan
+```
+
+</TabItem>
+
+</Tabs>
 
 ## Azure Pipelines
 
+To add Semgrep into your Azure Pipeline:
+
+1. Access the YAML pipeline editor within Azure Pipelines by following the [YAML pipeline editor](https://learn.microsoft.com/en-us/azure/devops/pipelines/get-started/yaml-pipeline-editor?view=azure-devops#edit-a-yaml-pipeline) guide.
+2. Copy the relevant code snippet provided after these instructions into the Azure Pipelines YAML editor.
```

Comment:
As previously, I am not sure why I have to remember for the next five steps what to do. If a user cannot do it now, it should be either the last step or our description of actions is not clear. Each step should be one action that the user can do at the same time.

Reply:
"After these instructions" refers to the layout of the text. I can't say "below" because Google Style Guide forbids it. (Same for the other two code snippets.) I'll make a header like you suggested.

Reply:
(Done making header for all CI providers.)